messaging/src/main/java/org/jboss/as/messaging/TransportConfigOperationHandlers.java

```diff
@@ -145,6 +145,16 @@ static void processConnectors(final OperationContext context, final Configuratio
                 final ModelNode config = property.getValue();
                 final Map<String, Object> parameters = getParameters(context, config);
                 final String clazz = FACTORY_CLASS.resolveModelAttribute(context, config).asString();
+                if (NettyConnectorFactory.class.getName().equals(clazz)) {
```

Comment:
Your patch is correct: you need to put the `socket-binding`'s name to the connector configuration. My remark is that you need to do that regardless of the value of the `factory-class`.